src/systemtest/java/com/ericsson/ei/systemtest/artifactflow/ArtifactFlowSteps.java

```diff
@@ -38,6 +41,13 @@ public void configurations_are_provided() {
 
     @Given("^a jenkins job '\\\"([^\\\"]*)\\\"' from '\"([^\"]*)\"' is created$")
     public void a_jenkins_job_from_is_created(String jenkinsJobName, String scriptFileName) throws Throwable {
+        /*JenkinsXmlData jenkinsXmlData = new JenkinsXmlData()
+                .addGrovyScript("")
+                .addBuildParameter("EVENT_ID")
+                .addBuildParameter("TEST_CASE_NAME");
+        
+        String test = jenkinsXmlData.getXmlAsString();*/
+        System.out.println();
```

Comment:
When @a-breid-ericsson is done with his PR in eiffel-commons, i will change the out commented lines. I won't merge like this :) Don't mind this part

Reply:
https://github.com/eiffel-community/eiffel-commons/pull/14